api/rest/restapi.go

```diff
@@ -855,64 +857,66 @@ func pinInfosToGlobal(pInfos []types.PinInfoSerial) []types.GlobalPinInfoSerial
 	return gPInfos
 }
 
-func sendResponse(w http.ResponseWriter, err error, resp interface{}) {
-	if checkErr(w, err) {
-		sendJSONResponse(w, 200, resp)
-	}
-}
+// sendResponse wraps all the logic for writing the response to a request:
+// * Write configured headers
+// * Write application/json content type
+// * Write status: determined automatically if given 0
```

Comment:
Not really against using `0` especially since the `net/http` status related constants start at 100 but due to the default value of an int being `0` makes me think that `-1` might be better, similar to how we use `-1` to indicate something special for replication factor. @hsanjuan totally up to you.

Reply:
yeah, I think -1 is better

Reply:
actually a local constant DefaultStatus is better